aws/data_source_aws_partition.go

```diff
@@ -15,10 +17,16 @@ func dataSourceAwsPartition() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
 			"dns_suffix": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"service_prefix": {
```

Comment:
Let's call this `reverse_dns_prefix` for clarity. 👍

Reply:
Done